docs/developer-cookbook.rst

```diff
@@ -447,34 +447,17 @@ the server root instead, and they may also want to move the Girder web client to
 be served out of an alternative route so they can still use it in addition to
 their custom front-end application.
```

Comment:
Since by default apps are not served at the root, we should probably change the section heading to something like "Mounting a custom application" and modify the first paragraph to something like

```
Mounting a custom application
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Normally, the root node (``/``) of the server will serve up the Girder web client.
A plugin may contain an entire application separate from the default Girder
web client. This plugin may be written in a way which enables administrators
to mount the application at a configured endpoint, including the option of
replacing the root node with the plugin application.
```